package.json

```diff
@@ -8,31 +8,23 @@
   },
   "devDependencies": {
     "@itsy/okay": "^0.0.4",
```

Comment:
Do we want to keep the `^` for `@itsy/okay`? :)

Reply:
fixed